client/src/components/controls/Controls.vue

```diff
@@ -62,6 +62,18 @@ export default defineComponent({
     function toggleEnhancements() {
       context.toggle('ImageEnhancements');
     }
+
+    const seekSegment = (direction: -1 | 1) => {
+      const segment = Math.floor((mediaController.frame.value - 150) / 450);
+      const frameMax = ((segment + 1) * 450) + 150;
+      const frameMin = (segment * 450) + 150;
+      if (direction > 0) {
```

Comment:
No need to make a change in this case, but just something to think about: since you're typing `direction` with a union of literals, you have the freedom to use more "readable" values here, perhaps `"backward" | "forward"`. Then line 70 becomes much more legible as `if (direction == "forward")`, ;as does the meaning of the function as a whole.